test/api/api.js

```diff
@@ -1,212 +1,232 @@
+/* eslint-env qunit */
+/* global videojs */
```

Comment:
I am actually going to define videojs at the top as `const videojs = window.videojs`